x/nft/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,583 @@
+package keeper_test
+
+import (
+	gocontext "context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/cosmos-sdk/x/nft"
+)
+
+func TestGRPCQuery(t *testing.T) {
+	suite.Run(t, new(TestSuite))
+}
+
+func (suite *TestSuite) TestGRPCQueryBalance() {
```

Comment:
No need to repeat `GRPCQuery` infix in all test names ... it's already set in the Suite name.